lib/grpc/stream.ex

```diff
@@ -128,6 +130,7 @@ defmodule GRPC.Stream do
 
     A `Flow` pipeline.
   """
+  @doc type: :transformer
```

Comment:
```suggestion
  @doc type: :transforms
```

let's use transforms as the tag instead of transformer